.github/workflows/build-linux-platforms.yml

```diff
@@ -1,47 +1,46 @@
 name: Build on Linux Platforms
 
+# Keep aligned with the options below
+env:
+  ALL_IMAGES: "['ubuntu:jammy', 'ubuntu:bionic', 'centos:7', 'rockylinux:8', 'debian:bullseye']"
+
 on:
   workflow_dispatch:
```

Comment:
Can be nice to trigger via label modification if possible as well.

Reply:
:+1: Agrees. This will be part of a later PR